src/pages/currency-trading-new/index.js

```diff
@@ -46,5 +46,9 @@ export default function CurrencyTrading() {
     openForm()
   }, [access])
 
-  return <>{plantId && access && <TransactionForm labels={_labelsADJ} access={access} plantId={plantId} />}</>
+  return (
```

Comment:
Why is this wrapped with a Box? This might break stuff.

Reply:
i need Zindex to prevent overlay

Reply:
can you move this to the transactionForm?